File: guides/micronaut-spring-boot/java/src/main/java/example/micronaut/Application.java
package example.micronaut;

import io.micronaut.runtime.Micronaut;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication // <1>
public class Application {

    public static void main(String... args) {
        Micronaut.run(Application.class);
    }
}

File: guides/micronaut-spring-web/kotlin/src/main/kotlin/example/micronaut/Application.kt
package example.micronaut

import io.micronaut.runtime.Micronaut.*
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object Application {

    @JvmStatic
    fun main(args: Array<String>) {    
        build()
            .args(*args)
            .packages("example.micronaut")
            .start()
    }
}
